src/cmd_build.rs

```diff
@@ -1,37 +1,33 @@
-use clap;
-
-use cargo_shim::{
-    Profile,
-    TargetKind
-};
+use cargo_shim::{Profile, TargetKind};
 
 use build::BuildArgs;
 use error::Error;
 
-fn command_build_or_check< 'a >( matches: &clap::ArgMatches< 'a >, should_build: bool ) -> Result< (), Error > {
-    let build_args = BuildArgs::new( matches )?;
+pub fn command_build_or_check<'a>(build_args: BuildArgs, should_build: bool) -> Result<(), Error> {
     let project = build_args.load_project()?;
 
-    let targets = project.target_or_select( |target| {
-        target.kind == TargetKind::Lib || target.kind == TargetKind::CDyLib || target.kind == TargetKind::Bin
+    let targets = project.target_or_select(|target| {
+        target.kind == TargetKind::Lib
+            || target.kind == TargetKind::CDyLib
+            || target.kind == TargetKind::Bin
     })?;
 
-    let config = project.aggregate_configuration( Profile::Main )?;
+    let config = project.aggregate_configuration(Profile::Main)?;
     for target in targets {
         if should_build {
-            project.build( &config, target )?;
+            project.build(&config, target)?;
         } else {
-            project.check( &config, target )?;
+            project.check(&config, target)?;
```

Comment:
Did you run `rustfmt` on this or something? Can you revert the purely formatting changes for lines which didn't change functionally, for this file and the others?

Reply:
yeah, my bad. will fix.